### PR TITLE
logout button navigation

### PR DESCRIPTION
**Pull request by reut201112**

### CODE REVIEW

**Comment by smartcity-hit** [comment]
logout icon needs to be as bellow:
image

but with your change:
image

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js [hunk 1]
```diff
@@ -111,9 +115,9 @@ const MenuAppBar = () =>{
     setOpenMenuAppBar(false);
   };
 
-  const handleMenuClose = () => {
-    setOpenCollapsed(false);
-  };
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:116]: why did you remove this function?

**Comment by reut201112** [comment]
created different pr and merged
